src/protocols/arweave/scaffold/manifest.js

```diff
@@ -0,0 +1,18 @@
+const source = ({ contract }) => `
+      owner: '${contract}'`
+
+const mapping = () => `
+      apiVersion: 0.0.5
+      language: wasm/assemblyscript
+      entities:
+        - ExampleEntity
```

Comment:
Does this entity exist in the example? 🤔

Reply:
good catch!, btw, is there any file we can place our example [schema.graphql](https://github.com/ChainSafe/subgraph-arweave-example/blob/master/schema.graphql)?

Reply:
I'm not sure I'm following, can you explain with a bit more detail please?

Reply:
If we generate a `Block` entity

```yml
 entities:
        - Block
```

in the template, developers can not run the template immediately since they don't have the `schema.gql` file

Reply:
I think you're probably referring to [this](https://github.com/graphprotocol/example-subgraph/issues/34). Basically all examples should be folders in the same repo, and the `schema.graphql` with the `Block` should be there.